tox.ini

```diff
@@ -206,114 +207,60 @@ setenv=
   ansible2.2: DELEGATE_FACTS_HOST = False
-  docker_cluster: PLAYBOOK = site-docker.yml.sample
-  docker_cluster_collocation: PLAYBOOK = site-docker.yml.sample
-  ooo_collocation: PLAYBOOK = site-docker.yml.sample
-  update_docker_cluster: PLAYBOOK = site-docker.yml.sample
-  purge_cluster_container: PLAYBOOK = site-docker.yml.sample
-  purge_cluster_container: PURGE_PLAYBOOK = purge-docker-cluster.yml
-  purge_lvm_osds_container: PLAYBOOK = site-docker.yml.sample
-  purge_lvm_osds_container: PURGE_PLAYBOOK = purge-docker-cluster.yml
-  add_osds: PLAYBOOK = site.yml.sample
-  add_osds_container: PLAYBOOK = site-docker.yml.sample
-  rgw_multisite: PLAYBOOK = site.yml.sample
-  rgw_multisite_container: PLAYBOOK = site-docker.yml.sample
-  shrink_mon_container: PLAYBOOK = site-docker.yml.sample
-  shrink_mon_container: MON_TO_KILL = mon2
-  shrink_osd_container: PLAYBOOK = site-docker.yml.sample
-  shrink_osd_container: COPY_ADMIN_KEY = True
-  shrink_osd: COPY_ADMIN_KEY = True
+  container: CONTAINER_DIR = /container
+  container: PLAYBOOK = site-docker.yml.sample
   container_podman: PLAYBOOK = site-docker.yml.sample
+  non_container: PLAYBOOK = site.yml.sample
+  container-purge_cluster: PURGE_PLAYBOOK = purge-docker-cluster.yml
+  shrink_mon: MON_TO_KILL = mon2
+  shrink_osd: COPY_ADMIN_KEY = True
 
   rhcs: CEPH_STABLE_RELEASE = luminous
-  luminous: CEPH_STABLE_RELEASE = luminous
-  luminous: CEPH_DOCKER_IMAGE_TAG = latest-luminous
-  luminous: CEPH_DOCKER_IMAGE_TAG_BIS = latest-bis-luminous
-  luminous: UPDATE_CEPH_STABLE_RELEASE = luminous
-  luminous: UPDATE_CEPH_DOCKER_IMAGE_TAG = latest
-  mimic: CEPH_STABLE_RELEASE = mimic
-  mimic: CEPH_DOCKER_IMAGE_TAG = latest-mimic
-  mimic: CEPH_DOCKER_IMAGE_TAG_BIS = latest-bis
-  mimic: UPDATE_CEPH_STABLE_RELEASE = mimic
-  mimic: UPDATE_CEPH_DOCKER_IMAGE_TAG = latest
   lvm_osds: CEPH_STABLE_RELEASE = luminous
-  lvm_osds: PLAYBOOK = site.yml.sample
-  lvm_osds_container: CEPH_STABLE_RELEASE = luminous
-  lvm_osds_container: PLAYBOOK = site-docker.yml.sample
-  bluestore_lvm_osds: CEPH_STABLE_RELEASE = luminous
-  bluestore_lvm_osds_container: CEPH_STABLE_RELEASE = luminous
-  bluestore_lvm_osds_container: PLAYBOOK = site-docker.yml.sample
-  lvm_batch_container: PLAYBOOK = site-docker.yml.sample
-  lvm_batch_container: CEPH_STABLE_RELEASE = luminous
+
+  rhcs: CEPH_STABLE_RELEASE = luminous
+  dev: CEPH_DOCKER_IMAGE_TAG = latest-master
+  dev: CEPH_DOCKER_IMAGE_TAG_BIS = latest-bis
+  dev: UPDATE_CEPH_DOCKER_IMAGE_TAG = latest
+  dev: UPDATE_CEPH_DEV_BRANCH = master
+  dev: UPDATE_CPEH_DEV_SHA1 = latest
   update_cluster: ROLLING_UPDATE = True
-  update_docker_cluster: ROLLING_UPDATE = True
-  add_osds: CEPH_STABLE_RELEASE = luminous
-  add_osds_container: CEPH_STABLE_RELEASE = luminous
-  rgw_multisite: CEPH_STABLE_RELEASE = luminous
-  rgw_multisite_container: CEPH_STABLE_RELEASE = luminous
+  update_cluster: CEPH_STABLE_RELEASE = mimic
+  update_cluster: CEPH_DOCKER_IMAGE_TAG = latest-mimic
```

Comment:
L228-230 => s/update_cluster/dev/

+ ensure it will deploy latest-master for every containers scenario but mimic for update_cluster container